recipe/recipe.yaml

```
@@ -1,24 +1,27 @@
-{% set name = "scikit-hep" %}
-{% set version = "2025.1.1" %}
-{% set python_min = "3.9" %}
+schema_version: 1
+
+context:
+  name: scikit-hep
+  version: 2025.1.1
+  python_min: 3.9
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name | replace('-', '_') }}-${{ version }}.tar.gz
   sha256: 7da189fabcbf657f7b273f8e15401f741b78f7e173c1c790c45704922ea27079
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: ${{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python {{ python_min }}
+    - python 3.9.*
```

```suggestion
    - python ${{ python_min }}.*
```

Currently there's a bug for all of these that the `python_min` doesn't get properly replaced everywhere, so you need to manually replace the `${{ python_min}}.*` in places like this.

c.f. https://github.com/conda-incubator/conda-recipe-manager/issues/307